general-contribution-guideline.md

```diff
@@ -0,0 +1,69 @@
+# Contribution Guidelines (general)
+
+This guide describes the process and the conventions for contributing an article without relaying too much on GitHub. For detailed instructions on how to contribute using GitHub, please refer to the [GitHub Contribution Guidelines](./github-contribution-guidelines.md). 
+
+## Announce
+
+1. **Check for Duplicates:** Is there already an article about your topic?
+  
+   * Please check our [Backlog of Articles](https://github.com/Zuehlke/zuehlke-book/issues?q=is%3Aissue+label%3Aarticle) whether there is already an article existing or planned for similar topic. You need a GitHub account for this.
+  
+   * If there is already a similar article, you can contact the author and find a solution (leave the subject to the author, offer your help as reviewer or co-author, take the subject over, or write a second article in response or extension to the first article, ...). 
+   
+2. **Announce an Article:**
+  
+   * Open a [New Issue for your Article](https://github.com/Zuehlke/zuehlke-book/issues/new?labels=article&title=Article:%20%3Cput%20topic%20title%20here%3E)
+  
+   * Describe roughly the idea you have in mind, what the article should be about.
+
+## Discuss
+
+It proved to be useful to discuss about an articles content before writing the article. Ask your colleagues, maybe they have some valuable input! 
+
+To discuss about an articles content, you have various options:
+
+   * Use the [Issues on Github](https://github.com/Zuehlke/zuehlke-book/issues): discuss directly in the issue you created - and don't miss to give feedback to other announced articles.
+
+   * Use Yammer, a coffee break, a team event or similar
+
+## Write
+
+**Content Conventions:**
+
+* written in English
+
+* short enough to be read in 10 minutes: size should be less than 2 A4 pages long in printout PDF version (that is between 400 and 750 words)
+
+* reflects on a single subject only (better make several articles, if you have too much material / ideas to write about)
+
+* matches our [Vision](./README.md#vision): it is about practicaly proven concepts, best practices, cultural philosophies or great ideas that we use in our daily work at Zühlke.
+
+* the subject is related to our work and the advice concrete enough to consider.
+
+* the subject is something you have experience on in some real projects and not just theoretical ideas
+
+**Language Conventions:**
+
+@Rolf: Please add title guideline
```

Comment:
@bruderol what was that link about the capitalized words in English titles?

Reply:
I not remember - I think I posted it to you somewhere. didnt I?

Reply:
thx